tutorials/pretraining-Vietnamese-data-curation/pretraining-Vietnamese-data-curation.ipynb

```diff
@@ -0,0 +1,1274 @@
+{
```

Comment:
Can you please rename the folder and notebook from `pretraining-Vietnamese-data-curation` to `pretraining-vietnamese-data-curation`?